### DISCUSSION TITLE
How do I ignore a specific file and a specific directory?

### DISCUSSION THREAD

**Discussion by kevinlinxc** [astral-sh/ruff#10675] [Q&A] [answered]
I can't use noqa because the file gets auto-generated every time we change something. So, it has to be an external way to ignore the file, kind of like gitignore. Something similar for ignoring entire directories too. I searched through the docs but couldn't find something like this, please let me know if it exists.

**Comment by kevinlinxc** [comment, marked as answer]
Ok sorry the terminology I needed was exclude instead of ignore:

https://docs.astral.sh/ruff/settings/#extend-exclude

**Reply by charliermarsh** [reply]
Yup, that sounds right to me.
